Fix: Add priority to FixHostname template

fh opened the pull request:
Fix: The FixHostname template currently does not transmit the log priority.

Note: There are several deprecation warnings in the specs, so I was unable to "just" add tests for this, and I considered changing the tests would blow this PR out of shape. PR for test setup is #22 - I rebased this PR on #22 to be able to test my changes, so preferably merge that one first

troy commented:
Nice! There's no space after the closing `>`) ([source](https://tools.ietf.org/html/rfc3164), particularly 5.4).

fh commented:
@troy Thanks, fix in a sec! Thats what one gets for the "no tests" statement, obviously.

troy commented:
You're welcome :) It looks like "timegenerated" became "timestamp." Was that intentional?

fh commented:
@troy Yeah, I got that stanza from Leon =)

fh commented:
@mheffner @troy I filed a separate PR to fix the testing setup - I would recommend to first merge that one, enable travis, and I'll add the test changes for this one?

troy commented:
@leonsodhi lgtm and I agree with @fh's proposed merge order.

fh commented:
I rebased the branch on #22, added the test, and found a minor bug (`<%` should be encoded).

fh commented:
Rebased to let travis run here, and somehow screwed up the rebase so that the `%` bugfix went MIA. Sorry, not sure how this happened. Fixed, pushed, now all green :)

mheffner commented:
Thanks!
